### PR TITLE
feat(route): add ielts

### LINKED ISSUE #9094: IELTS雅思最新消息
[Issuehunt badges](https://issuehunt.io/r/DIYgod/RSSHub/issues/9094)

### 类型

学习

### 网站地址

https://ielts.neea.cn/allnews?locale=zh_CN

### 网站描述

雅思中国新闻通知网站

### 需要生成什么内容？

·原文链接
·标题
·正文

### 额外描述

网站的消息顺序似乎并不是按照日期来排的，希望大佬们能按照时间顺序排列一下

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $5.00)

- [vfanghao vfanghao](https://issuehunt.io/u/vfanghao) ($5.00)

### Submitted pull Requests
- [#9217 feat: add ielts](https://issuehunt.io/r/DIYgod/RSSHub/pull/9217)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/DIYgod/RSSHub/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

### PR DESCRIPTION
**Pull request by zenxds**
## 该 PR 相关 Issue / Involved issue

Close #9094

## 完整路由地址 / Example for the proposed route(s)

```routes
/ielts
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [x] `Puppeteer`

## 说明 / Note

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#9094: IELTS雅思最新消息](https://issuehunt.io/repos/127769231/issues/9094)
---
</details>

### CODE REVIEW

**Review by SukkaW — CHANGES_REQUESTED** [review]
Please remove the modification of eslint configuration.

**Comment by zenxds** [comment]
> Please remove the modification of eslint configuration.

Done.

File: lib/v2/ielts/radar.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'ielts.neea.cn': {
+        _name: 'IELTS雅思',
+        '.': [
```
Comment (TonyRL) [inline lib/v2/ielts/radar.js:4]:
```suggestion
    'neea.cn': {
        _name: 'IELTS雅思',
        ielts: [
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/ielts/radar.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+            {
+                title: '雅思最新消息',
```
Comment (TonyRL) [inline lib/v2/ielts/radar.js:6]:
```suggestion
                title: '最新消息',
```
Title here should match the H3 header of the docs

**Pull request merged**